mackup/applications/mps-youtube.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = MPS Youtube
+
+[configuration_files]
+.config/mps-youtube/config
+.config/mps-youtube/playlist_v2
```

Comment:
It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).